packages/protobuf/messages.json

```diff
@@ -5577,6 +5577,96 @@
                 }
             }
         },
+        "NostrGetPubkey": {
```

Comment:
We probably don't want to update this here

Reply:
removed